mgmt_cli_test.py

```diff
@@ -1395,6 +1395,12 @@ def test_backup_and_restore_only_data(self):
         5. Run the verification read stress to ensure the data is restored correctly.
         """
         self.run_prepare_write_cmd()
+
+        compaction_ops = CompactionOps(cluster=self.db_cluster)
+        #  Disable keyspace autocompaction cluster-wide since we dont want it to interfere with our restore timing
+        for node in self.db_cluster.nodes:
+            compaction_ops.disable_autocompaction_on_ks_cf(node=node)
+
```

Comment:
That's something I missed during initial implementation.
Can be a good improvement for restore benchmark.
@Michal-Leszczynski What do you think?

Reply:
But we have to remember that after restore the SM will enable the autocompaction, so if we want to run and measure read stress then we have to disable it once again

Reply:
we don't collect any post-restore read stress metrics in these tests AFAIR

Reply:
I think it worth explicitly disabling as a precondition to the test. It reminds everyone about our assumptions, and makes no room for future mistakes.

Reply:
There is no such a thing. every ks created during the test will be not affected by whatever one has disabled before

Reply:
I argued in favor leaving explicit disable as you wrote them. Am I missing something?